src/test/kotlin/com/hibob/academy/service/UserServiceTest.kt

```diff
@@ -0,0 +1,152 @@
+package com.hibob.academy.service
+
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import org.mockito.Mockito.*
+import org.mockito.kotlin.any
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.whenever
+
+class UserServiceTest{
+    private val userDao= mock<UserDao>{}
+    private val notificationService= mock<NotificationService>{}
+    private val emailVerificationService= mock<EmailVerificationService>{}
+
+    private val userService=UserService(userDao,notificationService,emailVerificationService)
+
+    @Test
+    fun `test User service registerUser is userDao null`() {
```

Comment:
Test name is unclear
Should explain what it tests and what is the expected behaviour, for example:
registerUser should throw exception if user already exists